europe/czech_republic/prague.md

```diff
@@ -0,0 +1,5 @@
+# Prague
+
+Every Australian I've spoken to that has visited Europe has listed Prague as either their favourite place, or close to their favourite place. It must be worth a visit, right?
+
+Find out more [here](https://en.wikipedia.org/wiki/Prague)
```

Comment:
[`here` links are not very accessible](https://webaccess.berkeley.edu/ask-pecan/click-here). Update it accordingly.

Reply:
Do you just mean change what the hyperlink says? If so, done.